.credo.exs

```diff
@@ -1,130 +1,79 @@
-      # to the tuple.
-      #
-      # To disable a check put `false` as second element:
-      #
-      #     {Credo.Check.Design.DuplicatedCode, false}
-      #
-      checks: [
-        {Credo.Check.Consistency.ExceptionNames},
-        {Credo.Check.Consistency.LineEndings},
-        {Credo.Check.Consistency.MultiAliasImportRequireUse},
-        {Credo.Check.Consistency.ParameterPatternMatching},
-        {Credo.Check.Consistency.SpaceAroundOperators},
-        {Credo.Check.Consistency.SpaceInParentheses},
-        {Credo.Check.Consistency.TabsOrSpaces},
+%{configs: [
+  %{name: "default",
+    files: %{
+      included: ["lib/", "src/"],
+      excluded: [~r"/_build/", ~r"/deps/"]
+    },
+    requires: [],
+    check_for_updates: false,
+    color: true,
+    strict: true,
 
-        # For some checks, like AliasUsage, you can only customize the priority
-        # Priority values are: `low, normal, high, higher`
-        {Credo.Check.Design.AliasUsage, priority: :low},
+    # You can customize the parameters of any check by adding a second element
+    # to the tuple.
+    #
+    # To disable a check put `false` as second element:
+    #
+    #     {Credo.Check.Design.DuplicatedCode, false}
+    #
+    checks: [
+      {Credo.Check.Consistency.ExceptionNames},
+      {Credo.Check.Consistency.LineEndings},
+      {Credo.Check.Consistency.MultiAliasImportRequireUse},
+      {Credo.Check.Consistency.SpaceAroundOperators},
+      {Credo.Check.Consistency.SpaceInParentheses},
+      {Credo.Check.Consistency.TabsOrSpaces},
 
-        # For others you can set parameters
+      {Credo.Check.Design.AliasUsage, false},
+      {Credo.Check.Design.DuplicatedCode, excluded_macros: []},
 
-        # If you don't want the `setup` and `test` macro calls in ExUnit tests
-        # or the `schema` macro in Ecto schemas to trigger DuplicatedCode, just
-        # set the `excluded_macros` parameter to `[:schema, :setup, :test]`.
-        {Credo.Check.Design.DuplicatedCode, excluded_macros: []},
+      {Credo.Check.Design.TagTODO},
+      {Credo.Check.Design.TagFIXME},
 
-        # You can also customize the exit_status of each check.
-        # If you don't want TODO comments to cause `mix credo` to fail, just
-        # set this value to 0 (zero).
-        {Credo.Check.Design.TagTODO, exit_status: 2},
-        {Credo.Check.Design.TagFIXME},
+      {Credo.Check.Readability.FunctionNames},
+      {Credo.Check.Readability.LargeNumbers, false},
+      {Credo.Check.Readability.MaxLineLength, false},
+      {Credo.Check.Readability.ModuleAttributeNames},
+      {Credo.Check.Readability.ModuleDoc},
+      {Credo.Check.Readability.ModuleNames},
+      {Credo.Check.Readability.ParenthesesOnZeroArityDefs, false},
+      {Credo.Check.Readability.ParenthesesInCondition},
+      {Credo.Check.Readability.PredicateFunctionNames},
+      {Credo.Check.Readability.SinglePipe, false}, # Off for now
```

Comment:
I would like this to be on because I think single pipe (`var |> call()`) is much less clear than `call(var)`. Is this off for now because we are using it? Perhaps we can turn it on to force no-single pipe just for changes?

Reply:
I should have improved the comment. It turned it off because there are many places we need to fix up, and I thought that would be nicer as a separate change. I'll do it as part of this PR now though.